src/pages/components/tooltip/usage.mdx

```diff
@@ -16,16 +16,22 @@ clarity to a user.
 
 </PageDescription>
 
+<InlineNotification>
+
+**New update** We have removed standard tooltip and icon button tooltip as
+variants from this guidance. There’s no need to include a separate tooltip
+within the component. There are no breaking changes, but we are working on
```

Comment:
I also don't fully understand what you mean about it being within other components.